src/DV7toDV8.sh

```diff
@@ -273,7 +460,73 @@ do
         rm "$DV8_RPU_BIN" 
         rm "$DV8_BL_RPU_HEVC"
     fi
+    
+    # Check if remuxing was successful
+    if [[ -f "$tempDir/$mkvBase.DV8.mkv" ]]; then
+        echo "Moving converted files to final location..."
+        # Move the DV8 mkv to final location
+        mv "$tempDir/$mkvBase.DV8.mkv" "$mkvBase.DV8.mkv"
```

Comment:
And rather than repeating each of these names for the 2nd parameter, since we have the `targetDir` pushed onto the stack, what do you think about using this form for each of these `mv` statements:

`mv "$tempDir/$mkvBase.DV8.mkv" .`

?

I'm hard-coding the filename suffix there, but the thought about making them all variable still applies.